src/httpclient.c

```diff
@@ -125,6 +128,13 @@ static int save_http3_ticket_cb(ptls_save_ticket_t *self, ptls_t *tls, ptls_iove
     return 0;
 }
 
+static void add_header(const char* name, const char* value)
+{
+    req.headers[req.num_headers].name = h2o_iovec_init(name, strlen(name));
+    req.headers[req.num_headers].value = h2o_iovec_init(value, strlen(value));
+    ++req.num_headers;
+}
```

Comment:
I would not mind adding a function for this purpose, but if we are to, I would appreciate it if you could convert existing code to also use this function.

Reply:
Of course, done.